### DISCUSSION TITLE
How to register alignment styles in react-quill with nextjs

### DISCUSSION THREAD

**Discussion by nitink66** [vercel/next.js#16435] [Help]
How to register alignment styles in react-quill with nextjs could anyone please help me out.
[Stack overflow question link](https://stackoverflow.com/questions/63479292/how-to-register-alignment-styles-in-react-quill)

**Comment by jamesmosier** [comment]
To register a global style, you must put it in `pages/_app.js` ([docs](https://nextjs.org/docs/basic-features/built-in-css-support#adding-a-global-stylesheet)):

```jsx
import '../../node_modules/react-quill/dist/quill.snow.css'

export default function MyApp({ Component, pageProps }) {
  return <Component {...pageProps} />
}
```

There is an [RFC](https://github.com/vercel/next.js/discussions/15208) to allow this CSS import on a per-page basis. And also an [issue](https://github.com/vercel/next.js/issues/13282) that also talks about this.
